packages/desktop-client/src/components/mobile/budget/BudgetTable.jsx

```diff
@@ -225,8 +225,9 @@ function BudgetCell({
   const format = useFormat();
   const { showUndoNotification } = useUndo();
   const [budgetType = 'rollover'] = useSyncedPref('budgetType');
+  const modalBudgetType = budgetType === 'rollover' ? 'envelope' : budgetType;
```

Comment:
The new budget type names are 'envelope' and 'tracking'.  This fixes the default envelope budget but is still broken if you switch the budget to the tracking style.  Just replace 'budgetType' in the ternary with 'tracking' and you should be good.